three.c

```diff
@@ -0,0 +1,66 @@
+/* Can Parlar - scans 3 numbers and checks if they can make a valid triangleand what will be the type of the triangle */
+#include <stdio.h>
```

Comment:
you submitted with three warnings, which were due to parentheses of the boolean expressions